### ISSUE TITLE
Improve further the accessibility of tables on the "Numbers and Dates" page

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#2678]
The tables are not bad from an accessibility point of view bu they could be better.

These are the main areas to be considered and addressed:

- [ ] complexity of column headings
- [ ] lack of row headings

This is initially an issue for one page as described but the conclusions might prove useful for tables on other pages in due course.

Not a priority.

**Comment by AlOneill** [comment]
Working off-line for now.

**Comment by DeniseColbert** [comment]
@AlOneill is there any update on this?

**Comment by AlOneill** [comment]
Cannot see anything obvious to change: the page will be reviewed by the Accessibility team and the FR documentation committee in due course.

**Comment by stsccfr** [comment]
Closing since last comment was that no changes were needed
